dcap/types/src/certification_data.rs

```diff
@@ -0,0 +1,405 @@
+// Copyright (c) 2022 The MobileCoin Foundation
+
+//! This module provides the Certification Data type and it's logic
+
+use crate::quote3::{le_u16, le_u32};
```

Comment:
Bringing in le_u16 and le_u32 from `qutoe3.rs` is creating a circular dependency.
I'm up for suggestions on this.  One thought is moving these to a module in `mc-sgx-util`, since `mc-sgx-core` is also using nom and it could take advantage of non fallible versions.

Reply:
Yeah I'd suggest lifting these utility functions out of the `quote3.rs` module. Either that or just duplicating them here since they really don't do much.

Reply:
I could see them being generally-useful later, it might be nice to have them in mc-sgx-util.